configureBuild.sh

```diff
@@ -173,6 +173,8 @@ setRepository() {
     suffix="adoptium/jdk11u-fast-startup-incubator"
   elif [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_BISHENG}" ]]; then
     suffix="openeuler-mirror/bishengjdk-${BUILD_CONFIG[OPENJDK_CORE_VERSION]:3}"
+  elif [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_LOONGSON}" ]]; then
+    suffix="Panxuefeng-loongson/jdk8u"
```

Comment:
This seems to be coming from your repo, should it be from your company's repo?

Reply:
You're right. But the company code repo does not have a tag to meet the build requirements, using my own fork repo is only a temporary solution.

Reply:
OK, we'll likely have to hold off until we have that in place to make sure there's no MitM attacks on the binary blob before it gets installed on one of our agents.

Reply:
I have pushed a new tag into loongson/jdk8u. It is ready for use now.